wolfcrypt/src/port/Espressif/esp32_util.c

```diff
@@ -343,7 +356,7 @@ int ShowExtendedSystemInfo(void)
         return 0;
     }
 
-int esp_ShowExtendedSystemInfo()
+WOLFSSL_LOCAL int esp_ShowExtendedSystemInfo()
```

Comment:
WOLFSSL_LOCAL in the .c does nothing. WOLFSSL_LOCAL sets a hidden visibility but should only be used in the header. If you don't want the function to be used outside the .c then make it static.

Reply:
Also please use (void) in the declaration.

Reply:
Ok. I added those only as a reminder that the `WOLFSSL_LOCAL` is in the header for the respective function.

I will remove them in future PR's and  add a comment reminder instead about the visibility decorator tag.

Reply:
It doesn't harm anything and we aren't 100% elsewhere so I went ahead and accepted it as-is.